README.md

```diff
@@ -81,52 +90,47 @@ We have made AWS credentials available in the following file, with the appropria
+Include logging that lets the executor know how many of the records failed to join, and include a join error threshold above which the script will error out. Please motivate why you have selected the error threshold that you have. Please also log the time taken to perform the operations described, and within reason, try to optimise latency and computational resources used.
 
 ### 3. Descriptive Analytic Tasks (if applying for a Data Analyst Position)
 Please use the `sr_hex_truncated.csv` dataset to address the following.
 
 Please provide the following:
-1. Provide a visual answer to the question "which areas and request types should Electricity concentrate on to reduce the overall volume of their requests".
-2. Provide a working prototype dashboard for monitoring progress in reducing Electricity service request volume per area, and per type.
+1. An answer to the question "In which suburbs should Water and Sanitation concentrate their infrastructure improvement efforts?". Please motivate how you related the data provided to infrastructure issues.
+2. Provide a visual mock of a dashboard for the purpose of monitoring progress in applying the insights developed in (1). Please provide an explanation as to how the data provided would be used to realise what is contained in your mock.
  
-An Executive-level person should be able to read this report and follow your analysis without guidance.
+An Executive-level, non-specialist person should be able to read this report and follow your analysis without guidance.
 
 ### 4. Predictive Analytic Tasks (if applying for a Data Science Position)
-Please use `sr_hex.csv` dataset, only looking at requests from the `Water and Sanitation Services` department.
-
-Please chose two of the following:
-1. *Time series challenge*: Predict the weekly number of expected service requests per hex for the next 4 weeks.
-2. *Introspection challenge*: Reshape the data into number of requests, per type, per hex in the last 12 months. Chose a particular request type, or group of requests. Develop a model that predicts the number of requests of your selected type, using the rest of your data. Based upon the model, and any other analysis, determine the drivers of requests of that particular type(s).
-3. *Classification challenge*: Classify a hex as formal, informal or rural based on the data derived from the service request data.
+Using the `sr_hex.csv` dataset, please chose __two__ of the following:
+1. *Time series challenge*: Predict the weekly number of expected service requests per hex that will be created each week, for the next 4 weeks.
+2. *Introspection challenge*: Reshape the data into number of requests created, per type, per hex in the last 12 months. Chose a particular request type, or group of requests. Develop a model that predicts the number of requests of your selected type, using the rest of your data. Based upon the model, and any other analysis, determine the drivers of requests of that particular type(s).
```

Comment:
Might be worth slight re-phrasing
A) Reshape the data into number of requests created, per type, per hex in the last 12 months for a request type of your choice.
B) Develop a model that predicts the number of requests of your selected type
C) use this model to predict the number in A